styles/Index.module.scss

```
@@ -0,0 +1,258 @@
+@import 'styles/variables';
+
+/* 69px is the height of the navbar */
+// Hero Banner Styles
+.heroBanner {
+    min-height:calc(70vh - 69px);
+    height:fit-content;
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    justify-content: center;
+    line-height: 1.12;
+    width: 100%;
+}
+
+.heroButton {
+    font-size: 20px;
+    font-weight: 500;
+    max-width: 12em;
+    position: relative;
+    padding: 7.5px 30px;
+    margin:0;
+    border-radius: 30px;
+    background-color:$hot-pink;
+    border: none;
+}
+
+.heroButton:hover{background-color: hotpink;}
+
+.heroGrid {
+    padding: 0;
+    margin: 0;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    margin-top: 50px;
+    margin-bottom: 50px;
+    img {
+        max-width: 400px;
+        width: 90vw;
+        height: 100%;
+    }
+}
+
+.heroGridItem, .heroGridItemSplash {
+    margin-left: 66px;
+    margin-right: 66px;
+    max-width: 600px;
+    h1 {
+        margin-top: 15px;
+        position: relative;
+        font-size: 3rem;
+        font-weight: 500;
+        color: $black;
+        max-width: 500px;
+        font-family: 'Bungee';
+    }
+    p {
+        margin-bottom: 25px;
+        font-size: 1.2em;
+    }
+}
+
+@keyframes fadeUp {
+    0% {
+      transform: translateY(6rem);
+      opacity: 0;
+    }
+    50% {opacity: 1;}
+    100% {opacity: 0;}
+  }
+
+.heroIcon1, .heroIcon2, .heroIcon3 {
+    position: absolute;
+    margin-left: 14rem;
+    margin-top: -1rem;
+    animation: fadeUp 5s ease 0ms 1, fadeUp 9s ease 12000ms infinite;
+    opacity: 0;
+    font-family: 'Bungee';
+    color: $hot-pink;
+    border-radius: 10px;
+    padding: .3rem;
+}
+
+.heroIcon1 {animation-delay: 500ms;}
+.heroIcon2 {animation-delay: 3500ms;}
+.heroIcon3 {animation-delay: 6500ms;}
+.heroSplashContainer {padding: 2rem}
+
+.heroGridItemSplash {
+    width: 45vw;
+    background-image: url('/assets/img/graphics/terminal.png');
+    border-radius: 18px;
+    background-size:cover;
+    box-shadow:$light-cotton-candy -40px 40px 0px -3px, $light-strawberry -40px 40px;
+}
+
+// Hero Banner Footer
+.heroFooterBackground{margin-top: 2rem; width: 100%;}
```

For any css properties that have more than one attribute, I think you should put each on a new line.

It's nice when everything fits on one but it makes it slightly harder to read and understand what's going on.
```suggestion
.heroFooterBackground{
    margin-top: 2rem; 
    width: 100%;
}
```